subiquity/server/controllers/install.py

```diff
@@ -811,6 +811,15 @@ async def stop_unattended_upgrades(self):
 # a metered connection.
 Unattended-Upgrade::OnlyOnACPower "false";
 Unattended-Upgrade::Skip-Updates-On-Metered-Connections "true";
+
+# Set MinimalSteps to false to speedup install times. "true", the default, will
+# atmoize updates to enable interrupts between downloads, but this causes the
+# install to slow down considerably. This is a major bottleneck of the install.
+# Some testing shows a 3-4x speedup on the unattended-upgrades section of the
+# install when this is set to false. However, this will make interrupting
+# this section of the install process difficult. Set to true, only if you
+# need to interrupt unattended-upgrades, or otherwise require this behavior.
```

Comment:
```suggestion
# this section of the install process difficult.
```
The last sentence is entirely correct, but modifying the code on the running system is [non-obvious](https://github.com/canonical/subiquity/blob/main/scripts/bind-patch.sh)  I'd like to drop this sentence.